.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+## Description of the change
+
+> Description here
+
+## Related Issue
+
+Issue #ID
+
+## Checklist
+
+### Development
+
+- [ ] The changes have been covered with tests, if necessary.
+- [ ] You have a green build, with the exception of the flaky tests.
+- [ ] UI and JS files are fully tested, and the UI buttons work for all modules or those affected by your changes (e.g., solve and analyze buttons).
+- [ ] The network calls work for all modules or those affected by your changes (e.g., solving a problem).
+- [ ] The console messages are validated for all modules or those affected by your changes.
+
+### Code Review
+
+- [ ] This pull request includes an explicative title and description.
+- [ ] The GitHub issue is linked.
+
+### Merging
+
+- [ ] At least one other engineer has approved the changes.
+- [ ] After merging the change, the issue requires you to follow up with the user.
```

Comment:
What do you mean by this last point?

Reply:
The goal is to remind the owner to follow up with the user who opened the Issue. Let's say a user/customer finds a bug and files an issue. We should get back to him once the bug is fixed.

Reply:
In that case, I suggest to rephrase a bit: "After PR is merged, inform the reporter."

Reply:
Done!